framework/src/eventloop/eventloop_timer.c

```diff
@@ -0,0 +1,156 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <queue.h>
+#include <debug.h>
+#include <libtuv/uv.h>
+#include <eventloop/eventloop.h>
+
+#include "eventloop_internal.h"
+
+typedef struct timer_cb_s {
```

Comment:
Please split definition of structure and typedef. That is a violation of coding rule.